### DISCUSSION TITLE
Week 11 and 12 - Mostly complete...

### DISCUSSION THREAD

**Discussion by derekscholes** [USU-ITLS-x265-23-summer/class-repo#232] [General]
Honestly, the past few weeks have been rough as I have been studying for my next big exam. I have taken all day but I think I am officially done with content and most of organization. There is one big white space in my CV that I dont know how to fix and I need to update colors. Otherwise it is complete. Please feel free to take a look. I will do better at reaching out and commenting on other projects. 

[Completed Project](https://www.ginutritiontherapy.com)

Tasks:
- Polish it up
- Change colors

**Comment by CJ3315** [comment]
I wasn't able to view your project but I'm sure it looks great! What colors did you go with?

**Comment by audreyhki** [comment]
You've done great this semester! Don't be too hard on yourself :) Thankfully we're so close to being done and we'll be finished with this course a little early! 
Similar to @CJ3315 I also wasn't able to view your project, the link you provided wasn't working for me. I'd love to be able to see it so I'll keep an eye out to see if you give any update!

**Comment by beto0890** [comment]
When I click on your link opens a new window and I get a message saying "not found"

**Comment by derekscholes** [comment]
Sorry, I don't know what happened, but I updated the link and it should work now, if not, here is the site: https://www.ginutritiontherapy.com

**Comment by kassyoveson** [comment]
Derek, your project is amazing! It has gone above and beyond what was learned in class. You deserve extra credit for all the pages. The website looks professional. It provides good information, and it was interesting to read your about page. Well done, you should be very proud of what you have accomplished in this project.
